setup.py

```diff
@@ -58,8 +56,7 @@
         'nose',
         'httpretty',
         'mock',
-        'factory_boy',
-        ndg_httpsclient_version
```

Comment:
reference to issue: there's a link to the issue in the file. Also talked about it here (i think): https://quandl.slack.com/archives/core-product-team/p1458223827000678